manual/source/Tutorial/Algorithms/Alignment/PairwiseSequenceAlignment.rst

```diff
@@ -448,7 +448,7 @@ Assignment 5
     * A CIGAR string is a different representation of an alignment.
       It consists of a number followed by an operation.
       The number indicates how many consecutive operations of the same type are executed.
-      Operations can be <tt>M</tt> for match, <tt>S</tt> for mismatch, <tt>I</tt> for insertion and <tt>D</tt> for deletion.
+      Operations can be **M** for match, **S** for mismatch, **I** for insertion and **D** for deletion.
```

Comment:
There are two types of match notations. One is  "alignment match" and the other is "sequence match". "M" means match or mismatch in "alignment match" notation as @rrahn said.  (CIGAR has "=" for a match, and "x" for  a mismatch in "sequence match" notation.)

http://zenfractal.com/2013/06/19/playing-with-matches/